### DISCUSSION TITLE
need an shortcut for "latest release"

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#16532] [Mobile]
we need an option to add shortcut for "latest releases" of our favourite repositories, we can make shortcut for issues, pull requests and discussions, but why no shortcut for latest releases?

 ![Screenshot_2022-05-13-21-19-39-038_com.github.android.jpg](https://user-images.githubusercontent.com/94877880/168320999-566ae69f-30d5-42b2-aef6-d6f12ac7ec7a.jpg)

**Comment by mxie** [comment]
Thanks for your feedback @Nit-nit! Can you tell me more about how this feature would be used? What kind of information would you like to see about releases? What kind of releases would we show to you (are these for projects you manage?)?

**Reply by NitinIsFit** [reply]
Sorry for being 1 year late...

![Screenshot_2023-06-17-14-31-36-617_com.github.android.jpg](https://github.com/community/community/assets/94877880/68bdb342-1bb9-4766-9b7c-190e4b36dc17)

So far we are able to create shortcuts for issues, pull requests and discussions, but users are not able to make shortcuts for releases

**Reply by NitinIsFit** [reply]
Like I have chosen this repository "discordjs/discord.js" and i want to create a shortcut for "latest > oldest releases" or "oldest > latest releases" for this repo and when we click on the shortcut it open the list for all the releases for the repository which we have chosen to make shortcut of to keep track of the releases.
